dp-cluster-setup-utility.py

```diff
@@ -316,7 +316,7 @@ def __call__(self, url, code, data):
     if 200 <= code <= 299:
       return code, self._parse(data)
     else:
-      return UnexpectedHttpCode('Unexpected HTTP code: %d url: %s response: %s' % (code, url, data))
+      raise UnexpectedHttpCode('Unexpected HTTP code: %d url: %s response: %s' % (code, url, data))
```

Comment:
It was a minor bug, but it only effected the way how the error is presented. Since the Python http client converts http error codes to exceptions by default, it is unlikely that this branch is ever executed.